payload/builder.go

```diff
@@ -16,7 +16,7 @@ type aps struct {
 	Category         string      `json:"category,omitempty"`
 	ContentAvailable int         `json:"content-available,omitempty"`
 	MutableContent   int         `json:"mutable-content,omitempty"`
-	Sound            string      `json:"sound,omitempty"`
+	Sound            interface{} `json:"sound,omitempty"`
```

Comment:
Are you able to make this a sound struct similar to what I did for the alert struct. 

That way for consistency with the existing they could either use like this:

`payload.NewPayload().SoundName("..").SoundVolume("..")`

or like this:

`payload.NewPayload().Sound(map[string]interface{}{
..
})`